x-pack/plugins/features/server/routes/index.ts

```diff
@@ -37,7 +35,8 @@ export function defineRoutes({ router, featureRegistry, getLegacyAPI }: RouteDef
               request.query.ignoreValidLicenses ||
               !feature.validLicenses ||
               !feature.validLicenses.length ||
-              getLegacyAPI().xpackInfo.license.isOneOf(feature.validLicenses)
+              (context.licensing!.license.type &&
```

Comment:
Did you want the `!` or the `?` type here? Isn't `?` the safe one where instead of turning off the type assertion it will resolve all of this as null/undefined rather than causing a null pointer excetpion.

Reply:
`!` License is always available, otherwise `features` plugin won't start without `licensing` plugin. There is no way in our typings to mark a field as required currently.